.nvmrc

```diff
@@ -1 +1 @@
-lts/fermium
+lts/gallium
```

Comment:
While deploying to cloudflare pages it would pull the node version from here. fermium is a 14.x and some of our packages depend on >=16.x. So bumped it to gallium which is that.